apps/ui/nuxt.config.ts

```diff
@@ -40,7 +40,7 @@ export default defineNuxtConfig({
    *  instructions to create the current interface. See:
    *  https://nuxt.com/docs/guide/concepts/rendering#client-side-rendering
    **/
-  ssr: false,
+  ssr: true,
```

Comment:
Ensure all existing and third-party Vue components are SSR compatible to avoid rendering issues. [important]